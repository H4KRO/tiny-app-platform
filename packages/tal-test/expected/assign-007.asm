main()
  entry:
    Literal 0
    MakeRecord
    DeclareLocal      name: toto, mutable: false, hasInitialValue: true
    Pop               inBlock: false
    Local             name: toto
    Literal "value"
    SetAttribute      name: property, forceRender: true
    Pop               inBlock: false
    Local             name: toto
    Attribute         name: property
