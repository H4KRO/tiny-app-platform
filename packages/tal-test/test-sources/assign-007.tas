let toto = {}
set toto.property = "value"
toto.property