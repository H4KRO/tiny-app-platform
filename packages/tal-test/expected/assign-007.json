[
  {
    "kind": "DeclareLocal",
    "mutable": false,
    "name": "toto",
    "value": {
      "kind": "Record",
      "entries": []
    }
  },
  {
    "kind": "Assign",
    "address": {
      "kind": "Attribute",
      "key": "property",
      "value": {
        "kind": "Local",
        "name": "toto"
      }
    },
    "value": "value"
  },
  {
    "kind": "Attribute",
    "key": "property",
    "value": {
      "kind": "Local",
      "name": "toto"
    }
  }
]